sql_generators/mobile_kpi_support_metrics/templates/attribution_clients.schema.yaml

```diff
@@ -0,0 +1,29 @@
+{{ header }}
```

Comment:
I don't think the schema is supposed to have a query?

Reply:
sorry, copy and paste mistake. Updated.